scripts/about.js

```diff
@@ -1,6 +1,9 @@
-$(document).ready(function () {
-  const VERSION = chrome.runtime.getManifest().version
-  const DATE = new Date().getFullYear()
-  $('#version').text(VERSION)
-  $('#year').text(DATE)
-})
+//about.js
+
+// from 'utils.js'
+/*   global feedbackPageURL */
+const VERSION = chrome.runtime.getManifest().version
+const DATE = new Date().getFullYear()
+$('#version').text(VERSION)
+$('#year').text(DATE)
+$('#reviews-page').attr('href', feedbackPageURL)
```

Comment:
Add target='_blank' attribute also

Reply:
Thanks, was thinking of adding that in other links too.

Reply:
I was planning to add target='_blank' to all of the links in the About page, but if you want to, go ahead.

Reply:
I added it in the `about.html` file. Please review : )